semanticdb/integration/src/main/scala/example/Classes.scala

```diff
@@ -31,3 +31,25 @@ object N {
     local + 2
   }
 }
+
+object CaseClasses1 {
```

Comment:
I was thinking of adding a line or 3 to existing cases, otherwise it's really hard to see what got fixed.

Reply:
Added.

Reply:
I meant there we way too many cases and we don't actually need all of them. Especially that it's hard to make sure everything is working in this case.